app/javascript/tests/mocks/mockPatients.js

```diff
@@ -857,4 +857,146 @@ const mockPatient5 = {
   white: null,
 };
 
-export { blankMockPatient, mockPatient1, mockPatient2, mockPatient3, mockPatient4, mockPatient5 };
+const mockPatient6 = {
```

Comment:
why was another mock patient added? you should be able to tweak one of the existing ones to make it work for your tests.  that keeps the mocks manageable

Reply:
I was concerned about potentially breaking a different test that relied on the existing mock's DoB and made a new one for simplicity's sake. I can definitely just tweak an existing one and fix any tests that breaks